.ci/Jenkinsfile-compile

```diff
@@ -49,6 +49,7 @@ pipeline {
                      "cubepilot_cubeorange_default",
                      "cubepilot_cubeyellow_console",
                      "cubepilot_cubeyellow_default",
+                     "gumstix_fmu-v6u_default",
```

Comment:
Given this follows the fmu-v6u standard, could you rename this to px4_fmu-v6u_default?

Reply:
Absolutely, that's how it was named originally.  I'll rename and push now.

Reply:
Updated and pushed.